homeassistant/components/holiday/config_flow.py

```diff
@@ -0,0 +1,87 @@
+"""Config flow for Holiday integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from holidays import country_holidays, list_supported_countries
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_COUNTRY, CONF_NAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.selector import (
+    CountrySelector,
+    CountrySelectorConfig,
+    SelectSelector,
+    SelectSelectorConfig,
+    SelectSelectorMode,
+    TextSelector,
+)
+
+from .const import CONF_PROVINCE, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_NAME): TextSelector(),
```

Comment:
The description presented to the user for this step are "Name" and "Country" and i'm not exactly sure what we're asking the user to enter here. Is name needed or can it be omitted, and just set the title automatically based on the country name?

Reply:
I was thinking about that, but I wanted the users to have the possibility to set names so they can e.g. set it to the names of friends or collegues who live in different countries. I thought this would be more flexible for users, but it's fine for me to omit the name completely and set it based on country/subdivision.

Reply:
OK, in that case my thought is that given the integration is using unique ids, users are able to use the built in features from Home Assistant to rename the calendar or the entity id like they can with any other entity. Your use case makes a lot of sense, though it is not unique for this integration and can be achieved using the already supported method.